.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is being ignored. If media files are part of your project's deliverables or need to be versioned, you should remove this line. Otherwise, ensure that ignoring this directory is intentional.